packages/python-packages/api-stub-generator/tests/docstring_parser_test.py

```diff
@@ -56,6 +56,17 @@
 ~azure.dummy.datastream
 """
 
+docstring_param_typing_optional = """
+:param group: Optional group to check if user exists in.
+:type group: typing.Optional[str]
```

Comment:
@johanste - is this the docstring type formatting we're going to be encouraging from now on?
And this renders nicely in sphinx?

Reply:
We should verify this, but https://pypi.org/project/sphinx-autodoc-typehints/ should be able to handle this. In fact, one shouldn't have to specify the :type: directive in the docstring.